content/en/docs/tasks/traffic-management/egress/egress-control/index.md

```diff
@@ -363,7 +363,15 @@ Set the value of `values.global.proxy.includeIPRanges` according to your cluster
 
 #### IBM Cloud Kubernetes Service
 
-Use `--set values.global.proxy.includeIPRanges="172.30.0.0/16\,172.21.0.0/16\,10.10.10.0/24"`
+To see which CIDR is used in the cluster use `ibmcloud ks cluster get -c <CLUSTER-NAME>` and look for the `Service Subnet`. On very old clusters, this may not work and so using `kubectl get svc -o wide -A` would be a way to narrow down the CIDR from the original option of `"172.30.0.0/16,172.21.0.0/16,10.10.10.0/24"`.
+
+For example on newer clusters:
+{{< text bash >}}
+$ ibmcloud ks cluster get --cluster my-cluster | grep "Service Subnet"
+Service Subnet:                 172.21.0.0/16
+{{< /text >}}
+
+Use `--set values.global.proxy.includeIPRanges="172.21.0.0/16"`
```

Comment:
This is quite unclear. What does "original option" mean? Also, why do you show the example after changing the subject to explain what to do if it doesn't work?
```suggestion
To see which CIDR is used in the cluster use `ibmcloud ks cluster get -c <CLUSTER-NAME>` and look for the `Service Subnet`:

{{< text bash >}}
$ ibmcloud ks cluster get --cluster my-cluster | grep "Service Subnet"
Service Subnet:                 172.21.0.0/16
{{< /text >}}

Then use `--set values.global.proxy.includeIPRanges="172.21.0.0/16"`

{{< warning >}}
On very old clusters, this may not work so you can use `--set values.global.proxy.includeIPRanges="172.30.0.0/16,172.21.0.0/16,10.10.10.0/24"` or
use `kubectl get svc -o wide -A` to further narrow down the CIDR value for the setting.
{{< /warning >}}
```

Reply:
Thank you for your suggestion. I had updated.